Python/optimizer.c

```diff
@@ -919,24 +935,31 @@ uop_opt_dealloc(PyObject *self) {
 PyTypeObject _PyUOpOptimizer_Type = {
     PyVarObject_HEAD_INIT(&PyType_Type, 0)
     .tp_name = "uop_optimizer",
-    .tp_basicsize = sizeof(_PyOptimizerObject),
+    .tp_basicsize = sizeof(_PyUOpOptimizerObject),
     .tp_itemsize = 0,
     .tp_flags = Py_TPFLAGS_DEFAULT | Py_TPFLAGS_DISALLOW_INSTANTIATION,
     .tp_dealloc = uop_opt_dealloc,
 };
 
 PyObject *
-PyUnstable_Optimizer_NewUOpOptimizer(void)
+PyUnstable_Optimizer_NewUOpOptimizer(int jit)
```

Comment:
Hi, maybe `jit` parameter can be declared as bool type?

Reply:
I don't think we `#include <stdbool.h>` in public headers (this function is declared in `Include/cpython/optimizer.h`).